compiler/record-minmax/driver/Driver.cpp

```diff
@@ -114,36 +123,20 @@ int entry(const int argc, char **argv)
   std::string input_data_format("h5");
   uint32_t num_threads = 1;
 
-  if (arser["--min_percentile"])
-    min_percentile = arser.get<float>("--min_percentile");
-
-  if (arser["--num_threads"])
-    num_threads = arser.get<int>("--num_threads");
-
+  ::get_values_from<float>(arser, "--min_percentile", 1.0);
```

Comment:
return value is not used. This will not update variables.

```suggestion
  float min_percentile = ::get_values_from<float>(arser, "--min_percentile", 1.0);
```

If you want to set default value with this function, it would be possible to remove the above default value setting.